cli/.angular-cli.json

```diff
@@ -0,0 +1,106 @@
+{
+    "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+    "project": {
+      "name": "ng-universal-demo"
+    },
+    "apps": [
+      {
+        "root": "src",
+        "outDir": "dist",
+        "assets": [
+          "assets",
+          "favicon.ico"
+        ],
+        "index": "index.html",
+        "main": "main.ts",
+        "polyfills": "polyfills.ts",
+        "test": "test.ts",
+        "tsconfig": "tsconfig.app.json",
+        "testTsconfig": "tsconfig.spec.json",
+        "prefix": "app",
+        "styles": [
+          "styles.css"
```

Comment:
@Toxicable, What do you think to make the app use SASS (or LESS)?
or do you think it's better to keep it as simple as possible?

In case you're going to use SASS, you can use `ng set defaults.styleExt scss` then you also need to change import styles (both for client and server) and also needs to change extensions from `.css` to `.scss/.sass`

Reply:
while it's totally possible for this to handle a css preprocessor I think it's best to keep this as an example of what the CLI can do with Universal rather than what the CLI can do

Reply:
Also don't want to be bias in this regard